src/main/scala/com/github/codelionx/dodo/discovery/ODCandidateQueue.scala

```diff
@@ -82,7 +82,7 @@ class ODCandidateQueue private(
 
   def queueLength: Int = queueSize
 
-  def pendingSize: Int = pendingOdCandidates.size
+  def pendingSize: Int = pendingOdCandidates.reduce((a, b) => a._2.size + b._2.size) + toBeStolenOdCandidates.reduce((a, b) => a._2.size + b._2.size)
```

Comment:
This does not compile as you can see in CI.

`reduce`  must return the same type as its arguments (and its a super type of the collection type): [`def reduce[B >: A](op: (B, B) => B): B`](https://www.scala-lang.org/api/current/scala/collection/Seq.html#reduce[B%3E:A](op:(B,B)=%3EB):B)

- You may use [`def foldLeft[B](z: B)(op: (B, A) => B): B`](https://www.scala-lang.org/api/current/scala/collection/Seq.html#foldLeft[B](z:B)(op:(B,A)=%3EB):B) here. It takes an initial value `z` and an operator function `op` that gets the accumulator (initialized with `z`)  and each element as arguments.
  ```scala
  pendingOdCandidates.foldLeft(0)( (sum, elem) => sum + elem._2.size )
  ```
- Or you can transform the sequence first, before reducing it:
  ```scala
  pendingOdCandidates.map(_._2.size).sum
  ```

> Code examples not tested, so they may contain syntax errors.